src/sdl2_compat.c

```diff
@@ -2211,7 +2211,7 @@ SDL_LoadWAV_RW(SDL2_RWops *rwops2, int freesrc, SDL2_AudioSpec *spec2, Uint8 **a
         SDL_RWops *rwops3 = RWops2to3(rwops2);
         if (rwops3) {
             SDL_AudioSpec spec3;
-            const int rc = SDL3_LoadWAV_RW(rwops3, freesrc ? SDL_TRUE : SDL_FALSE, &spec3, audio_buf, audio_len);
+            const int rc = SDL3_LoadWAV_RW(rwops3, freesrc, &spec3, audio_buf, audio_len);
```

Comment:
This should probably be `(freesrc != 0)`

Reply:
Changed it and all its similars